lib/reporters/mini.js

```diff
@@ -11,9 +11,9 @@ var repeating = require('repeating');
 var colors = require('../colors');
 
 chalk.enabled = true;
-Object.keys(colors).forEach(function (key) {
+for (let key of Object.keys(colors)) {
 	colors[key].enabled = true;
-});
+}
```

Comment:
Ah, oki :) I will commit with updates based on your review, but without doing anything regarding the loop.

I did a quick try where I changed how we create mini and verbose reporters in their test setup. Making sure to include {colors: true} as an options. That way we can use your suggestion:
```
chalk.enabled = this.options.color;
for (const key of Object.keys(colors)) {
	colors[key].enabled = this.options.color;
}
```

It works for the mini and verbose tests themselves, but it caused this test to fail because chalk and colors did not seem to work as intended, https://github.com/avajs/ava/blob/master/test/cli.js#L73